unit_tests/web_tests/web_test_page_objects.py

```diff
@@ -472,6 +472,11 @@ def vm_overlay_column_is(self, row, column, data):
         element = self.driver.find_element_by_xpath(xpath)
         return element.text == str(data)
 
+    def checkbox_selected(self, checkbox_id):
```

Comment:
This method exists again below in a different class as "is_checkbox_selected." It should be moved to the super-class (Page) so there is only a single definition and can be used by both sub-classes.